collector.go

```diff
@@ -0,0 +1,77 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package datadog
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"go.opencensus.io/stats/view"
+)
+
+const rate = float64(1)
+
+// client implements datadog.Client
```

Comment:
s/client/collector

Also, the text says that it implements "datadog.Client" but I see no package named "datadog" being imported. Is it supposed to be "statsd.Client"?